Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -2929,9 +3235,15 @@ def fetch_incidents(client: Client, fetch_limit: int, first_fetch: str, fetch_th
 def threat_to_incident(threat) -> dict:
     threat_info = threat.get('threatInfo', {}) if IS_VERSION_2_1 else threat
     incident = {
-        'name': f'Sentinel One Threat: {threat_info.get("classification", "Not classified")}',
-        'occurred': threat_info.get('createdAt'),
-        'rawJSON': json.dumps(threat)}
+        "name": f'Sentinel One Threat: {threat_info.get("classification", "Not classified")}',
+        "labels": [
+            {"type": _type, "value": value if isinstance(value, str) else json.dumps(value)}
+            for _type, value in threat.items()
+        ],
+        "details": json.dumps(threat),
```

Comment:
When are the labels added and why are they needed here?
The `details` looks exactly like `rawJSON`, why do you need the duplication?

Reply:
> I think that it is not right to reopen the incident again and again every time there is a status change which is not "resolved".
In addition, I don't see where you take into account the real status when mirroring in- you don't set it to be "in_progress" for example.

Here we tried to achieve the exact sync between the remote(SentinelOne) and XSOAR platforms so we are reopening it whenever the status is other than "resolved". If you think this is not a good approach to do then we will remove that entire reopening of the incident mechanism from the code

Reply:
> When are the labels added and why are they needed here? The `details` looks exactly like `rawJSON`, why do you need the duplication?

we took the other mirroring integration which is suggested by one of the reviewers. they are also populating the labels and details after doing that thing in our integration it shows the investigation data and labels in the incident. so we decided to add these two things to our incidents which are very helpful.
![image](https://user-images.githubusercontent.com/82433049/222064851-1bcede30-858f-489b-9b0a-3e2e2d91fd88.png)